BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Content/NewContentFormView.js

```diff
@@ -35,11 +33,11 @@ class NewContentFormView extends mix(AbstractFormView).with(FormViewButtonsMixin
         let template = this._renderTemplate('Content/contentEditView', {
             contentTypeId: this._contentTypeId,
             language: this._language,
-            name: this._name,
```

Comment:
@alavieille : to avoid that shit of passing the title from views to views or sometimes even via request ...
The final view is responsible to find and display its title. Two ways to do this : wait for the form to be fetch (what I choose in this PR) or do an API call to get the information, but it's not performant as you'd have to make 2 calls (1 for API, 1 for the form)